codegen/XCTestPlans/ProtocolTestPlan.xctestplan

```diff
@@ -0,0 +1,143 @@
+{
+  "configurations" : [
+    {
+      "id" : "13F35EEE-6A97-4B33-90EE-D251A1D42E6F",
+      "name" : "Test Scheme Action",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "environmentVariableEntries" : [
+      {
+        "key" : "AWS_DEFAULT_REGION",
+        "value" : "us-west-2"
+      },
+      {
+        "key" : "AWS_REGION",
+        "value" : "us-west-2"
+      },
+      {
+        "key" : "AWS_ACCESS_KEY_ID",
+        "value" : "your-access-key-id"
+      },
+      {
+        "key" : "AWS_SECRET_ACCESS_KEY",
+        "value" : "your-secret-access-key"
+      },
+      {
+        "key" : "AWS_SESSION_TOKEN",
+        "value" : "your-session-token"
+      },
+    ]
+  },
```

Comment:
Protocol Test Plan contains fake credentials. This is required for github actions simulators